### PR TITLE
Updating SKR policy to include TDX Attestation Type

### PR DESCRIPTION
**Pull request by jvega10**
Updating SKR Policy to include new TDX Attestation Type.
- [X] Validated SNP CVM CMK using the new policy
- [x] Validated TDX CVM CMK using new policy

### CODE REVIEW

File: cvm_deployment/key/skr-policy.json [hunk 1]
```diff
@@ -2,117 +2,189 @@
...
+              "equals": "tdxvm"
+            },
+            {
+              "claim": "x-ms-attestation-type",
+              "equals": "sevsnpvm"
+            }
+          ]
         }
       ],
       "authority": "https://shareditn.itn.attest.azure.net/"
```
[the remark this answers is not in the document]
Comment (jvega10) [reply cvm_deployment/key/skr-policy.json:190]: Just added all the known MAA endpoints!

File: cvm_deployment/key/skr-policy.json [hunk 2]
```diff
@@ -3,119 +3,464 @@
...
+            },
+            {
+              "claim": "x-ms-attestation-type",
+              "equals": "sevsnpvm"
+            }
+          ]
+        }
+      ],
+      "authority": "https://sharedjpw.jpw.attest.azure.net/"
     }
```
Comment (canfikret) [inline cvm_deployment/key/skr-policy.json:464]: missing: sharedsan.san.attest.azure.net, shareduaen.uaen.attest.azure.net, sharedwus2.wus2.perf.attest.azure.net

Comment (canfikret) [reply cvm_deployment/key/skr-policy.json:464]: MAA team confirmed that these 2 endpoints not to be included.

**Review by canfikret — APPROVED** [review]

File: cvm_deployment/key/skr-policy.json [hunk 3]
```diff
@@ -3,119 +3,464 @@
...
+            },
+            {
+              "claim": "x-ms-attestation-type",
+              "equals": "sevsnpvm"
+            }
+          ]
         }
       ],
-      "authority": "https://sharedsasia.sasia.attest.azure.net/"
+      "authority": "https://sharedeus2e.eus2e.attest.azure.net/"
```
Comment (canfikret) [inline cvm_deployment/key/skr-policy.json:232]: sharedeus2e.eus2e is duplicated?

Comment (jvega10) [reply cvm_deployment/key/skr-policy.json:232]: I only see one

**Review by thaiolivia — APPROVED** [review]

**Pull request merged**